requirements.txt

```diff
@@ -1,26 +1,26 @@
-Django>=1.8,<1.9
+Django==1.10.5
```

Comment:
As discussed: I prefer to manually update (so we know when the django version is behind) and in slack we get notifications when django updates. Just remember to do a version update on dienst then